static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -0,0 +1,210 @@
+    }
+}
+
+.new-header__nav {
+    @include fs-textSans(5);
+    display: flex;
+    align-items: center;
+
+    @include mq($until: phablet) {
+        padding-left: ($gs-gutter / 2);
+        padding-right: ($gs-gutter / 2);
+        // Position after pseudo element which forces line break
+        order: 1;
+        flex: 1;
+        justify-content: space-between;
+    }
+
+    // TODO: Line heights?
+    @include mq(tablet) {
+        font-size: 19px;
+    }
+
+    @include mq(desktop) {
+        font-size: 22px;
+    }
+}
+
+.new-header__nav__menu-button-and-user-button {
+    display: flex;
+    align-items: center;
+
+    @include mq($until: phablet) {
+        margin-left: 4px;
+    }
+}
+
+.new-header__nav__menu-button {
+    // Override button from user agent stylesheet
+    background-color: colour(news-main-2);
+    // Unset button from user agent stylesheet
+    border: 0;
+    // Override button from user agent stylesheet
+    // Must be square
+    padding: $gs-gutter * .6;
+    border-radius: 100%;
+    z-index: $zindex-modal;
+
+    @include mq($until: phablet) {
+        // Overlap
+        margin-top: -5px;
+        margin-bottom: -5px;
+    }
+
+    @include mq(phablet) {
+        margin-left: 6px;
+    }
+
+    svg {
+        height: 18px;
+        // Removes extra spacing
+        display: block;
+        fill: #ffffff;
+    }
+}
+
+.new-header__nav__profile-button {
+    // Unset button from user agent stylesheet
+    border: 0;
+    // Unset button from user agent stylesheet
+    padding: 0;
+    // Override button from user agent stylesheet
+    background-color: colour(guardian-brand-dark);
+    border-radius: 100%;
+    // Mask contents to keep them in the circle
+    overflow: hidden;
+
+    @include mq($until: phablet) {
+        margin-left: 3px;
+    }
+
+    @include mq(phablet) {
+        order: -1;
+    }
+
+    svg {
+        // Removes extra spacing
+        display: block;
+        box-sizing: border-box;
+        // Spacing above icon
+        padding-top: 3px;
+        width: 20px;
+        height: 20px;
+
+        path {
+            fill: colour(news-support-1);
+        }
+    }
+}
+
+.new-header__nav__list {
+    display: flex;
+    flex-wrap: wrap;
+    // Unset ul from _lists.scss
+    margin-left: 0;
+    // Unset ul from user agent stylesheet
+    margin-top: 0;
+    // Unset ul from user agent stylesheet
+    margin-bottom: 0;
+
+    @include mq($until: phablet) {
+        order: -1;
+    }
+
+    @include mq(phablet) {
+        margin-left: $gs-gutter / 2;
+    }
+}
+
+.new-header__nav__list-item {
+    // Override inherited nav ul from _lists.scss
+    list-style: none;
+    white-space: nowrap;
+
+    &:not(:last-child) {
+        &:after {
+            content: '/';
+            color: colour(news-main-2);
+        }
+    }
+}
+
+.new-header__nav__link {
+    // Override a from _lists.scss
+    color: #ffffff;
+    padding: 2px;
+    // Override a from user agent stylesheet
```

Comment:
Ah! These comments have the format: `<Override/unset> <selector> from <selector location>`, so in this case I'm saying “this style overrides another which is on an `a` selector”.

If you're interested [I wrote something a while back](https://oliverjash.me/2014/11/20/comment-your-css-relationships) on why I find these helpful. Essentially, when you read CSS it doesn't tell you why, so that's what I'm trying to do with these comments. If the code tells you why it's there, you can more easily reason about it when refactoring later. Often times, I find undocumented CSS gets left around because no-one dares to touch it!

In any case, I plan to remove these comments once iteration has finished, because this is not a practice we follow as a team.